### PR TITLE
[netatmo] Adding smoke detector module

### PR DESCRIPTION
**Pull request by clinique**

### CODE REVIEW

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
Normally, features for OH 3.3 are already frozen but considering my relative good knowledge of the code of this binding, I see no risk of regression with this PR and so I would be in favour to merge it. @jlaur: can you have a look yourself and confirm you agree with me ?

File: bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml [hunk 1]
```diff
@@ -382,6 +390,18 @@
 				<option value="MOVEMENT_HUMAN">Human seen</option>
 				<option value="MOVEMENT_VEHICLE">Car seen</option>
 				<option value="MOVEMENT_ANIMAL">Animal seen</option>
+				<option value="SOUND_TEST_OK">Alarm test successfull</option>
```
Comment (jlaur) [inline bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml:393]:
```suggestion
				<option value="SOUND_TEST_OK">Alarm test successful</option>
```

File: bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml [hunk 2]
```diff
@@ -382,6 +390,18 @@
...
+				<option value="SOUND_TEST_ERROR">Alarm test failed</option>
+				<option value="DETECTOR_READY">Smoke detector installed</option>
+				<option value="DETECTOR_TAMPERED">Smoke detector tampered</option>
+				<option value="DETECTION_CHAMBER_CLEAN">Detection chamber clean</option>
+				<option value="DETECTION_CHAMBER_DIRTY">Detection chamber dusty</option>
+				<option value="BATTERY_LOW">Battery low</option>
+				<option value="BATTERY_VERY_LOW">Battery very low</option>
+				<option value="SMOKE_CLEARED">Smoke cleared</option>
+				<option value="SMOKE_DETECTED">Smoke detected</option>
+				<option value="WIFI_STATUS_OK">Wifi status ok</option>
```
Comment (jlaur) [inline bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml:403]:
```suggestion
				<option value="WIFI_STATUS_OK">Wi-Fi status ok</option>
```

File: bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml [hunk 3]
```diff
@@ -382,6 +390,18 @@
...
+				<option value="WIFI_STATUS_ERROR">Wifi status error</option>
```
Comment (jlaur) [inline bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml:404]:
```suggestion
				<option value="WIFI_STATUS_ERROR">Wi-Fi status error</option>
```

File: bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml [hunk 4]
```diff
@@ -357,6 +357,14 @@
 				<option value="ACCEPTED_CALL">Call is incoming</option>
 				<option value="INCOMING_CALL">Call has been answered by a user</option>
 				<option value="MISSED_CALL">Call has not been answered by anyone</option>
+				<option value="HUSH">Smoke detector status</option>
+				<option value="SMOKE">Smoke detection</option>
+				<option value="TAMPERED">Smoke Detector tamper</option>
+				<option value="WIFI_STATUS">Wifi status</option>
```
Comment (jlaur) [inline bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml:363]:
```suggestion
				<option value="WIFI_STATUS">Wi-Fi status</option>
```

**Review by jlaur — CHANGES_REQUESTED** [review]
LGTM. Only some minor text issues.

File: bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml (hunk 3 above)
Comment (clinique) [reply bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml:404]: Corrected

File: bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml (hunk 4 above)
Comment (clinique) [reply bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml:363]: Corrected

**Review by jlaur — APPROVED** [review]
Nice last-minute addition! :-)

**Pull request merged**